src/api/app/controllers/trigger_controller.rb

```diff
@@ -28,9 +28,12 @@ def rebuild
   def release
     raise NoPermissionForPackage.setup('no_permission', 403, "no permission for package #{@pkg} in project #{@pkg.project}") unless policy(@pkg).update?
 
-    matched_repo = @pkg.project.repositories.includes(:release_targets).any? { |repo| matched_repo?(repo) }
+    manual_release_targets = @pkg.project.release_targets.where(trigger: 'manual')
+    raise NoPermissionForPackage.setup('not_found', 404, "#{@pkg.project} has no release targets that are triggered manually") unless manual_release_targets.any?
 
-    raise NoPermissionForPackage.setup('not_found', 404, "no repository from #{@pkg.project} could get released") unless matched_repo
+    manual_release_targets.each do |release_target|
+      release_package(@pkg, release_target.target_repository, @pkg.release_target_name, release_target.repository, nil, nil, nil, true)
+    end
 
```

Comment:
Will this still report an error if no package could be released? (not that I really care, but it would be a change in behaviour)

Reply:
the first package that encounters an exception (can only be missing permissions or something unforseen) will stop the action. see the test.